src/marqo/tensor_search/models/custom_vector_object.py

```diff
@@ -0,0 +1,24 @@
+from marqo.tensor_search.enums import MappingsObjectType
+from marqo.tensor_search.constants import ALLOWED_CUSTOM_VECTOR_CONTENT_TYPES
+
+
+def custom_vector_schema(index_model_dimensions: int) -> dict:
+  return {
+    "$schema": "http://json-schema.org/draft-04/schema#",
```

Comment:
You should use a pydantic model for this. These approach to validation is not very readable and intuitive